readme.md

```diff
@@ -116,7 +116,7 @@ Once installation is working, take a look at the existing code to make sure you
 
 * **Question:** Why is it important to give a non-specific error message as opposed to a message like "Password incorrect?"
 
-* **Your Answer:** 
+* **Your Answer:** It's important because we want to keep that error message internal.
```

Comment:
Specifically to provide less information to malicious users trying to brute force entry.